templates/bareos-dir/storage/storage.conf.j2

```diff
@@ -4,6 +4,8 @@ Storage {
   Address = "{{ item.bareos_dir_ip }}"
   Password = "{{ bareos_storage_password }}"
   Media Type = {{ item.media_type | default('File') }}
+{% for dev in item.device %}
   Device = {{ item.device }}
-  Maximum Concurrent Jobs = 50
+{% endfor %}
+  Maximum Concurrent Jobs = {{ item.conc_jobs | default('50') }}
```

Comment:
Could it be `maximum_concurrent_jobs`?

Reply:
Done

Reply:
My bad, I just noticed in another PR that the role already uses `max_concurrent_jobs`. I also see several variables starting with `max_` in the READMe. So `max_concurrent_jobs` might be more accurate.

Reply:
Done